SSI.php

```diff
@@ -55,6 +55,10 @@
 if (substr($sourcedir, 0, 1) == '.' && substr($sourcedir, 1, 1) != '.')
 	$sourcedir = dirname(__FILE__) . substr($sourcedir, 1);
 
+// For php below 7
+if (!function_exists('random_int'))
+	require_once ($sourcedir . '/random_compat/random_int.php');
+
```

Comment:
Instead of making a new folder and file, can we just dump this into Subs-Combat.php?  Its the intended purpose.

Reply:
FYI, also moving it into Subs-Compat should fix the build failing issue.